cookbooks/aws-parallelcluster-platform/kitchen.platform-config.yml

```diff
@@ -5,6 +5,14 @@ verifier:
     - test
 
 suites:
+  - name: sticky_bits
```

Comment:
We should remove this part from the old configs (there are 2 occurrences).

Reply:
done